### PR TITLE
EZEE-2772 Calendar widget

### PR DESCRIPTION
**Pull request by michalmilc90**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | [EZEE-2772](https://jira.ez.no/browse/EZEE-2772)
| Versions      | master

Adds a section on the calendar widget.

### CODE REVIEW

File: docs/publishing/advanced_publishing_options.md [hunk 1]
```diff
@@ -49,3 +51,68 @@
...
+    |![Today button](img/calendar_widget_today.png)|Display the current day regardless of the view.|
+    |![Arrows](img/calendar_widget_arrows.png)|Navigate through months, weeks, or days.|
+    |![Toggle month/week/day](img/calendar_widget_toggler_mwd.png)|Switch between the month, week, or day view.|
+    |![Toggle calendar/list](img/calendar_widget_toggler_cal_list.png)|Switch between the calendar and list view,|
+    |![Filters](img/calendar_widget_filters.png)|Access and apply calendar filters.|
+    
+    ### Calendar filters
+        
+    With numerous events appearing in the calendar widget, you can declutter the view by applying filters.
+    You can filter by the events' type or modified language in all three — month, week, and day — views.
```
Comment (DominikaK) [inline docs/publishing/advanced_publishing_options.md]:
```suggestion
    You can filter by the events' type or modified language in all three views (month, week, and day).
```

File: docs/publishing/advanced_publishing_options.md [hunk 2]
```diff
@@ -49,3 +51,68 @@
...
+    #### Rescheduling or cancelling events
+    
+    In case of publishable events (e.g. articles), you can change or cancel their planned publication by using the toolbar buttons.
+    
+    |Button|Description|
+    |------|-----------|
+    |![Reschedule button](img/selection_action_bar_reschedule.png)|Reschedule all selected events.|
+    |![Discard button](img/selection_action_bar_discard.png)|Cancel the future publication of selected events.|
+    
+    Select all events to perform a bulk action on them.
```
Comment (DominikaK) [inline docs/publishing/advanced_publishing_options.md]:
```suggestion
    Select all events you want to perform a bulk action on.
```

File: docs/publishing/advanced_publishing_options.md [hunk 3]
```diff
@@ -49,3 +51,68 @@
...
+    Using the toolbar buttons, perform one of the following actions:
+    
+    - To change the publication time, click the **Reschedule** button.
+    In the **Reschedule** content modal window, select the new date and click **Confirm reschedule**.
```
Comment (DominikaK) [inline docs/publishing/advanced_publishing_options.md]:
```suggestion
    In the **Reschedule** modal window, select the new date and click **Confirm reschedule**.
```
Unless "Reschedule content" is the full button name?

**Comment by michalmilc90** [comment]
> It's fine for me.
> 
> We could also mention about pagination/load more in the list view, but I know @tischsoic is working on the pagination for the week view and day view so maybe you want to handle this all at once.

That's a very good point @dew326, thank you! I will put the PR on hold and update it when the pagination is done and merged.

**Pull request merged**